test_unstructured/chunking/test_title.py

```diff
@@ -30,7 +30,8 @@ def test_it_rejects_max_characters_not_greater_than_zero(max_characters: int):
     elements: List[Element] = [Text("Lorem ipsum dolor.")]
 
     with pytest.raises(
-        ValueError, match=f"'max_characters' argument must be > 0, got {max_characters}"
+        ValueError,
+        match=f"'max_characters' argument must be > 0, got {max_characters}",
```

Comment:
@Coniferish I think your local Black settings might be off because these changes are not required by our Black configuration and in fact not desired. If parameters fit on a single line they are left that way (without a trailing comma), which conserves vertical space (how much will fit in a single screenful of code).